docs/pages/project/contributing/contributing-error.md

```diff
@@ -142,7 +142,7 @@ var (
 func main() {
```

Comment:
@Aviral0702 Good job utilizing the loglevel here, but do you know what it is? Is it a variable or a string?

If it is a variable, does it have a value assigned to it?
Answer: Refer to the example provided under the issue.